CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ and this project adheres to
   `mem_size_mib` and `track_dirty_pages` being mandatory for all
   `PATCH /machine-config` requests. Now, they can be omitted which leaves these
   parts of the machine configuration unchanged.
+- [#5007](https://github.com/firecracker-microvm/firecracker/pull/5007): Fixed
+  watchdog softlockup error when a vcpu is paused for GDB.
```

Comment:
nit: "... is paused while debugging with GDB."

Reply:
Also maybe we should specify that the error happens inside the guest?

Reply:
we should also mention that it is x86-specific.

Reply:
Resolved thanks